README.md

```diff
@@ -1,50 +1,43 @@
 # Universal Permissionless Rewards Distributor
 
-A universal permissionless rewards distributor written in Solidity. It allows the distribution of any reward token (different reward tokens are possible simultaneously) based on a Merkle tree distribution, and using ERC20 allowance of a treasury.
+A universal permissionless rewards distributor written in Solidity.
```

Comment:
Mostly putting each sentences on its own line, so that future changes are clearer